### PR TITLE
R4R:add the caculate task about account rewards and validator tokens

### PR DESCRIPTION
**Pull request by weichang-bianjie**

### CODE REVIEW

File: backend/orm/document/account.go [hunk 1]
```diff
@@ -68,6 +68,27 @@ func (a Account) GetAccountList() ([]Account, error) {
 	return result, err
 }
 
+func (a Account) GetRewardsAccount(offset, size int) ([]Account, error) {
```
Comment (kaifei-bianjie) [inline backend/orm/document/account.go:71]: rename to `GetDelegatores`

File: backend/orm/document/validator_static.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+)
+
+const (
+	CollectionNameExValidatorStatic = "ex_validator_static"
+
+	ValidatorStaticFieldOperatorAddress = "operator_address"
+	ValidatorStaticFieldDate            = "date"
+)
+
+type ExValidatorStatic struct {
```
Comment (kaifei-bianjie) [inline backend/orm/document/validator_static.go:18]: add `status` field

Comment (weichang-bianjie) [reply backend/orm/document/validator_static.go:18]: have done

File: backend/orm/document/types.go [hunk 3]
```diff
@@ -17,5 +17,6 @@ var (
 
 	Docs = []Doc{
 		TaskControlModel,
+		new(ExStaticRewards),
```
Comment (kaifei-bianjie) [inline backend/orm/document/types.go:20]: add `ExValidatorStatic` to Docs

**Pull request merged**